spec/pdc/v1/global_component_contact_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PDC do
```

Comment:
I think this should be "describe PDC::V1::GlobalComponentContac" so that all these tests in the same label, like following:
PDC::V1::ContentDeliveryRepo::count
  test_0001_has serveral cdn content delivery repos               PASS (0.00s)

Reply:
Agreed